docs/csharp/language-reference/keywords/floating-point-types-table.md

```diff
@@ -1,26 +1,30 @@
 ---
-title: "Floating-Point Types Table (C# Reference)"
-ms.date: 07/20/2015
+title: "Floating-point types table (C# Reference)"
+description: "Overview of the built-in C# floating-point types"
+ms.date: 08/20/2018
 helpviewer_keywords: 
   - "floating-point numbers [C#]"
   - "ranges of floating-point types [C#]"
   - "types [C#], floating-point types"
 ms.assetid: da886cc5-e01e-4f62-b3ec-6428c8f7a102
 ---
-# Floating-Point Types Table (C# Reference)
+# Floating-point types table (C# Reference)
+
 The following table shows the precision and approximate ranges for the floating-point types.  
   
 |Type|Approximate range|Precision|  
 |----------|-----------------------|---------------|  
 |[float](float.md)|±1.5e−45 to ±3.4e38|7 digits|  
 |[double](double.md)|±5.0e−324 to ±1.7e308|15-16 digits|  
+|[decimal](decimal.md)|(-7.9 x 10<sup>28</sup> to 7.9 x 10<sup>28</sup>) / (10<sup>0</sup> to 10<sup>28</sup>)|28-29 digits|  
```

Comment:
Unrelated to your PR, since the range is the same as in the [decimal topic](https://review.docs.microsoft.com/en-us/dotnet/csharp/language-reference/keywords/decimal), but it's confusing. I think it's preferable to include a more simple form: -7.9228 x 10\<sup>28\<\sup> to 7.9228 x 10 \<sup>28\<\sup>, which is the value reported in the [Decimal](https://docs.microsoft.com/en-us/dotnet/api/system.decimal?view=netframework-4.7.2) documentation. If you make that change, it would also be good to change the range in the decimal keyword topic.

Reply:
@rpetrusha what do you think of providing a range in the format which is used for `float` and ` double`: from `+/- <closest to zero>` to `+/- <maximum absolute value>`: ±1 x 10<sup>-28</sup> to ±7.9228 x 10<sup>28</sup>

Reply:
I think that that's fine, @pkulikov.